hasura/metadata/databases/default/tables/public_nominees.yaml

```diff
@@ -139,6 +139,19 @@ event_triggers:
     headers:
       - name: verification_key
         value_from_env: HASURA_EVENT_SECRET
+  - name: createNomineeDiscordBot
+    definition:
+      enable_manual: false
+      insert:
+        columns: '*'
+    retry_conf:
+      interval_sec: 10
+      num_retries: 5
+      timeout_sec: 60
+    webhook: '{{HASURA_API_BASE_URL}}/event_triggers/eventManager'
+    headers:
+      - name: verification_key
+        value_from_env: HASURA_EVENT_SECRET
```

Comment:
@topocount is this usually added manually or should hasura add this to the yaml file automatically?